### DISCUSSION TITLE
Steam Wishlist Demo Feature

### DISCUSSION THREAD

**Discussion by SilvertongueRED** [SteamDeckHomebrew/decky-loader#503] [Ideas]
Was thinking it would be swell for someone to add this feature to the Deck's steam wishlist so we can easily see and then download games on our list with demos! Here's a link to the person who made a chrome extension for inspiration! 

https://chrome.google.com/webstore/detail/demos-in-my-steam-wishlis/cfdhaglgnhlacodkjjmlbcglgecnelnj

**Comment by TrainDoctor** [comment]
Plugin suggestions are not a accepted usage of the discussions. Thanks regardless.
